EpiAware/docs/src/getting-started/tutorials/intro.md

```diff
@@ -0,0 +1,66 @@
+The diagram below shows the relationship between the modules in the package for a typical workflow.
```

Comment:
I think this mermaid diagram needs updated? Maybe make an issue for that.

Reply:
yup it does. Once this is merged going to make a wave of issues for all the doc gaps